### PR TITLE
Update Economics Major (ECO)

### PR DESCRIPTION
**Pull request by richardhuangz**

### CODE REVIEW

File: tigerpath/majors_and_certificates/majors/ECO.json [hunk 1]
```diff
@@ -77,16 +77,19 @@
...
...ir sophomore year and earn a letter grade of C or better. MAT 201 or EGR 156 can be used as a substitute for MAT 175 for entry into the department.\n\nStudents will only be exempt from the Economics Department’s MAT 175 pre-requisite if they have already taken an acceptable college level course in multivariate calculus and earned a grade of C or better. Please note that the AP BC calculus curriculum is not sufficient since it only teaches univariate calculus, while MAT 175 focuses on multivariate calculus.\n\nStudents who wish to take math-track econometrics (ECO 312), upper level finance certificate courses (such as ECO 462, ECO 465 and ECO 466), or pursue graduate studies in economics and finance should take MAT 201 and MAT 202 instead of MAT 175. Both courses should be taken for a letter grade by this subset of students. The finance certificate program strongly advises that you take both MAT 201 and MAT 202; only the highest of the two grades is recorded for the purpose of fulfilling the entry requirements. Students who choose to fulfill this requirement by taking the one-semester MAT 175 will be responsible for mastering the linear algebra part of the curriculum on their own.",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/ECO.json:80]: https://economics.princeton.edu/undergraduate-program/prospective-concentrators/entrance-prerequisites/

File: tigerpath/majors_and_certificates/majors/ECO.json [hunk 2]
```diff
@@ -77,16 +77,19 @@
...
           "req_list": [
             {
               "name": null,
               "max_counted": 1,
-              "min_needed": null,
+              "min_needed": 1,
               "explanation": null,
               "course_list": [
-                "MAT 175"
+                "MAT 175",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/ECO.json:89]: https://economics.princeton.edu/undergraduate-program/prospective-concentrators/entrance-prerequisites/

"MAT 201 or EGR 156 can be used as a substitute for MAT 175 for entry into the department."

File: tigerpath/majors_and_certificates/majors/ECO.json [hunk 3]
```diff
@@ -185,6 +188,7 @@
           "course_list": [
             "COS 445",
             "ELE 381",
+            "ENE 422/MAE 422",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/ECO.json:191]: https://economics.princeton.edu/undergraduate-program/course-selection/pre-approved-cognates/

This page applies to all changes below.

Comment (churichard) [reply tigerpath/majors_and_certificates/majors/ECO.json:191]: I think we're missing POL 432 and POL 505.

Comment (richardhuangz) [reply tigerpath/majors_and_certificates/majors/ECO.json:191]: Gotcha , fixed.

**Review by churichard — APPROVED** [review]

**Pull request merged**